js/ui/appletManager.js

```diff
@@ -580,6 +580,15 @@ function createApplet(extension, appletDefinition, panel = null) {
         // so we don't need to pass it to the constructor anymore, but would
         // require a compatibility clean-up effort.
         applet = module.main(extension.meta, orientation, panel.height, applet_id);
+
+        //set applet style class
+        let metadata = extension.meta;
+        let appletClass = '';
+        if(typeof metadata === 'object' && metadata !== null && typeof metadata.uuid === 'string'){
+            appletClass = ' ' + metadata.uuid.toLowerCase().replace(/([^a-z0-9]+)/gi, '-') + '-applet';
+        }
+        applet._setStyleClass('applet-box'+appletClass);
```

Comment:
What exactly is the purpose of prepending 'applet-box' **and** appending 'applet'? I'd say one of them is sufficient.

Reply:
applet-box is the default class name and because i use a set method its overwrited so this, is in the _init of Applet class 
when creating the actor is setted to applet-box and is a common class in a lot of theme so i don't think is possible remove it, for the appending applet is done only if we have an uuid and is appendend to the parsed uuid (note the space  before it)

Reply:
so the custom class name ends up in something like menu-cinnamon-org-applet is more cosmetic but i thinks is more intuitive